.github/workflows/android-perf.yml

```diff
@@ -298,15 +298,25 @@ jobs:
               --workflow-run-attempt ${{ github.run_attempt }}
           done
 
-          ls -lah benchmark-results
-
-          for BENCHMARK_RESULTS in benchmark-results/*.json; do
-            cat "${BENCHMARK_RESULTS}"
-            echo
+          for SCHEMA in v2 v3; do
+            for BENCHMARK_RESULTS in benchmark-results/"${SCHEMA}"/*.json; do
+              cat "${BENCHMARK_RESULTS}"
+              echo
+            done
           done
 
-      - name: Upload the benchmark results
+      # TODO (huydhn): Remove v2 schema once the benchmark dashboard finishes the migration
+      - name: Upload the benchmark results (v2)
+        uses: pytorch/test-infra/.github/actions/upload-benchmark-results@main
+        with:
+          benchmark-results-dir: benchmark-results/v2
+          dry-run: false
+          schema-version: v2
```

Comment:
Isn't `GITHUB_TOKEN` required by v2?

Reply:
Oh, this token is used to query the job ID, which isn't available as part of the job context.  In the old schema, it is supplied by `.github/scripts/extract_benchmark_results.py` script using the handmade regex in https://github.com/pytorch/executorch/blob/main/.github/scripts/extract_benchmark_results.py#L24.  In the new version, the action uses the token to do it on behalf of the caller, so `.github/scripts/extract_benchmark_results.py` would only need to extract the benchmark results from AWS, and there is no need to use any hardcoded logic.  I'll eventually clean up the script later.